robocop/run.py

```diff
@@ -83,6 +86,11 @@ def log_message(self, **kwargs):
 
     def load_checkers(self):
         checkers.init(self)
+        if self.config.list:
+            for checker in self.checkers:
+                for msg in checker.messages:
+                    print(checker.messages[msg])
```

Comment:
How its printed, it not require some basic sorting? Or it doesnt matter

Reply:
It's printed by using `__str__` magic method which is a part of PR #71 : 
``` python
def __str__(self):
        return f'Message - {self.msg_id} [{self.severity.value}]: {self.name}: {self.desc}'
```
So it needs to be merged before this PR 😃